lock/src/test/java/org/infinispan/lock/ClusteredLockSplitBrainTest.java

```diff
@@ -88,11 +86,13 @@ public void testLockUseAfterPartitionWithoutMajority() throws Throwable {
       ClusteredLock lock5 = clm5.get(LOCK_NAME);
 
       splitCluster(new int[]{0, 1, 2}, new int[]{3, 4, 5});
+      partition(0).assertDegradedMode();
+      partition(1).assertDegradedMode();
```

Comment:
Please add a comment here, something like "Wait for degraded topologies to work around ISPN-9008"

Reply:
ok